docs/source/data-model.rst

```diff
@@ -139,13 +139,19 @@ A typical example
     Time is given in UNIX time (seconds since 1970). Software for looking at
     the data would, of course, translate that into a more human-readable form.
 
-Projections (Experimental):
-
-The run_start metadata can include a projections field.
-It is intended that a projection is an aid to interacting with external systems using standardized vocabularies.
-Projections might be used in a variety of use cases such as providing run data to analysis tools or suitcases.
-Each projection represents multiple ways to represent data from the run. Each field in the projection dictionary 
-is an unique and externally-identifiable string and each value is an instruction for accessing dat from the run. 
+.. _projections:
+
+Projections (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Run Start document may include a ``projections`` field.  It is intended
```

Comment:
Two spaces used after periods in this paragraph, consider 's/  / '

Reply:
@tacaswell has a long explanation of double blank spaces. I think it was discussed during one of the Pilot calls.

Reply:
https://en.wikipedia.org/wiki/Sentence_spacing_in_language_and_style_guides 🤯

Reply:
https://www.cultofpedagogy.com/two-spaces-after-period/

Reply:
I would argue that "Nothing Says Over 40 Like Two Spaces after a Period!" should be corrected to "Nothing Says Over 
50 Like Two Spaces after a Period!" 😆